### ISSUE TITLE
The loss is going up after adding ELMo

### ISSUE THREAD

**Issue by rzhangpku** [allenai/allennlp#3250]
**Describe the bug**
After I replace the BiLSTM in the "Input Encoding" component of ESIM with ELMo, but the loss on the training set is going up. Why is the loss on the training set going up? 

Here is the code of using Elmo:
```
allennlp.modules.elmo.Elmo(options_file, weight_file, 1, requires_grad=True, dropout=self.dropout)
```

**Expected behavior**
The loss on the training set is going down.

**System (please complete the following information):**
 - OS: Ubuntu
 - Python version: Python 3.6.8
 - AllenNLP version: "I installed from master"
 - PyTorch version: 1.1

**Comment by DeNeutoy** [comment]
Hi, typically we didn't find that fine tuning elmo helps for downstream tasks and makes training very unstable. I'd recommend that you try with `requires_grad=False` instead.

**Comment by rzhangpku** [comment]
Yes, when I try with `requires_grad=False`, the loss on the training set will not go up. Thanks a lot!
